### PR TITLE
Issue #650: another block of fixes for WinAPI types

### PR DESCRIPTION
**Pull request by airelil**
Switch to wintypes in handleprops and common_controls.
Explicit signatures for WinAPI functions used by RemoteMemoryBlock

### CODE REVIEW

File: pywinauto/handleprops.py [hunk 1]
```diff
@@ -331,8 +331,8 @@ def enum_child_proc(hwnd, lparam):
     # define the child proc type
     enum_child_proc_t = ctypes.WINFUNCTYPE(
         ctypes.c_int,            # return type
-        win32structures.HWND,    # the window handle
-        win32structures.LPARAM)  # extra information
+        ctypes.wintypes.HWND,    # the window handle
+        ctypes.wintypes.LPARAM)  # extra information
```
Comment (vasily-v-ryabov) [inline pywinauto/handleprops.py]: The same concern.

File: pywinauto/win32functions.py [hunk 2]
```diff
@@ -180,14 +196,50 @@
...
-VirtualAlloc		=	ctypes.windll.kernel32.VirtualAlloc
-VirtualFree			=	ctypes.windll.kernel32.VirtualFree
-WriteProcessMemory	=	ctypes.windll.kernel32.WriteProcessMemory
+VirtualAlloc = ctypes.windll.kernel32.VirtualAlloc
+VirtualAlloc.restype = wintypes.LPVOID
+VirtualAlloc.argtypes = [
+    wintypes.LPVOID,
+    ctypes.c_size_t,
+    wintypes.DWORD,
+    wintypes.DWORD
```
Comment (vasily-v-ryabov) [inline pywinauto/win32functions.py]: (minor) Comma can be after every element to simplify potential copy-paste or `Ctrl+D`.

File: pywinauto/remote_memory_block.py [hunk 3]
```diff
@@ -108,7 +108,7 @@ def __init__(self, ctrl, size=4096):
         self._as_parameter_ = self.mem_address
 
         # write guard signature at the end of memory block
-        signature = win32structures.LONG(0x66666666)
+        signature = ctypes.wintypes.LONG(0x66666666)
```
[the remark this answers is not in the document]
Comment (airelil) [reply pywinauto/remote_memory_block.py]: Interesting! I wasn't aware about it. I guess, it depends on python distributions. Will change it.

File: pywinauto/win32functions.py [hunk 4]
```diff
@@ -180,14 +196,50 @@
...
+    wintypes.DWORD,
+]
+WriteProcessMemory = ctypes.windll.kernel32.WriteProcessMemory
+WriteProcessMemory.restype = wintypes.BOOL
+WriteProcessMemory.argtypes = [
+    wintypes.HANDLE,
+    wintypes.LPVOID,
+    wintypes.LPVOID,
+    ctypes.c_size_t,
+    ctypes.POINTER(ctypes.c_size_t)
```
Comment (vasily-v-ryabov) [inline pywinauto/win32functions.py]: (minor) One more comma is missing. The rest is good!

**Comment by vasily-v-ryabov** [comment]
This pull request **introduces 2 alerts** when merging 8fe8d648b0a4eeb24d640bce9c01e2fb2dd7a367 into 3e72fdf3488e97ecf907dc2b7c7762e7c48a8030 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-5bfbf0c126f201b652e6a19554666362a472e4a0)

**new alerts:**

* 2 for Module is imported with &#39;import&#39; and &#39;import from&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by vasily-v-ryabov** [comment]
This pull request **introduces 1 alert** and **fixes 10** when merging f88e025362af64a52fefa4f03903a5e8d7cf4bc9 into 3e72fdf3488e97ecf907dc2b7c7762e7c48a8030 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-2ff9ab5c8dc6e15f45175a21b24809c425e62132)

**new alerts:**

* 1 for Module is imported with &#39;import&#39; and &#39;import from&#39;

**fixed alerts:**

* 10 for Variable defined multiple times

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 10 alerts** when merging 6f9617c5d6a1d8578321ac1ad6c9003209ba26aa into 3e72fdf3488e97ecf907dc2b7c7762e7c48a8030 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-2208148a658a7b7d909fcc5418f4e4ad29bb5af9)

**fixed alerts:**

* 10 for Variable defined multiple times

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**